vagrant/README.md

```diff
@@ -145,7 +145,7 @@ NAME      mTLS
 default   on
 ```
 
-If you try to access the [marketplace](http://192.168.33.20:8080), it won't work because everything is encrypted.
+If you try to access the [marketplace](http://127.0.0.1:8080), it won't work because everything is encrypted.
```

Comment:
That should not be the case.

`127.0.0.1:8080` on a `host` is port forwarded directly to `http-server` listening on `127.0.0.1:8080` inside a `guest`.

This traffic is not going through the dataplane, therefore it won't be affected by enabling mTLS.

Instead, you should say `If you try to access http://192.168.33.20:18080, it won't work` (because that traffic does go through a dataplane)